docs/Installation.md

```diff
@@ -227,15 +227,39 @@ Make sure Zonemaster-GUI is properly installed.
 3. Verify that when you mouse over this text the versions of the following
    Zonemaster components are shown: Backend, Engine and GUI.
 
-
-
 ## What to do next?
 
  * For a JSON-RPC API, see the Zonemaster::Backend [JSON-RPC API] documentation.
  * For a command line interface, follow the [Zonemaster::CLI installation] instruction.
  * For a Perl API, see the [Zonemaster::Engine API] documentation.
  * For Https, see [Let's Encrypt / Certbot].
 
+## Serving the GUI and API from a custom base url
```

Comment:
This section requires to have the three steps that should be done.

Reply:
done